packages/tests/tests/filter/single_node/subscribe.node.spec.ts

```diff
@@ -219,11 +220,14 @@ describe("Waku Filter V2: Subscribe", function () {
     });
   });
 
-  it("Subscribe to 30 topics at once and receives messages", async function () {
-    const topicCount = 30;
+  it("Subscribe to 100 topics at once and receives messages", async function () {
```

Comment:
I think it is better to split this test in two: one for legacy versions of nwaku with testing 30 topics and another for `0.24.0` with 100 topics. 

In this way it will be less misleading considering the name of the test.

Reply:
@fbarbu15

Reply:
like this? https://github.com/waku-org/js-waku/pull/1803